src/formatter.js

```diff
@@ -26,7 +27,7 @@ assign(Formatter.prototype, {
   // a "joining" value is specified (e.g. ' and ')
   parameterize(values, notSetValue) {
     if (typeof values === 'function') return this.parameter(values);
-    values  = Array.isArray(values) ? values : [values];
+    values  = isArray(values) ? values : [values];
```

Comment:
It is actually no change. That translates to:

```js
const { isArray } = Array;
// compiles to:
var isArray = Array.isArray;
```

See [object destructuring](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).